packages/devtools_app/lib/src/code_size/code_size_controller.dart

```diff
@@ -339,6 +375,14 @@ extension CodeSizeJsonFileExtension on DevToolsJsonFile {
     return false;
   }
 
+  bool get isV8Snapshot {
```

Comment:
can we use the isV8Snapshot utility from package:vm_snapshot_analysis

Reply:
that's weird. Have you debugged this locally to see what's happening?

Reply:
Looks like I was in fact not passing in the expected data. It's working now.